README.md

```diff
@@ -1 +1,36 @@
-# Mobile.github.io
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Portfolio Page
+
+> This is the mobile version of a personal portfolio page that shows of my accomplishments as a software developer.
+
+![screenshot](images/Capture.PNG)
+
+You can access to my project, via this link: https://zuhra-hashimi.github.io/Mobile.github.io/
```

Comment:
[OPTIONAL] What do you think about just having [live demo](https://zuhra-hashimi.github.io/Mobile.github.io). You can also do it. Look at the way I did it in the screenshot. No need for the full stop. 😃

![image](https://user-images.githubusercontent.com/86200356/193766787-8f429061-a132-49c8-a209-dad1ca6d1b4a.png)